interop/kotlinx-metadata/src/main/kotlin/com/squareup/kotlinpoet/metadata/classinspectors/ElementsClassInspector.kt

```diff
@@ -333,8 +332,7 @@ public class ElementsClassInspector private constructor(
 
     val propertyData = declarationContainer.properties
       .asSequence()
-      .filter { it.isDeclaration }
-      .filterNot { it.isSynthesized }
```

Comment:
No need for this `filterNot` anymore?

Reply:
Nope, one thing learned in this is that this is mutually exclusive with declaration/delegation/etc

Reply:
Nice